apisix/discovery/nacos.lua

```diff
@@ -307,11 +311,21 @@ local function fetch_full_registry(premature)
             return
         end
 
+        if not up_apps[namespace_id] then
+            up_apps[namespace_id] = {}
+        end
+
+        if not up_apps[namespace_id][group_name] then
+            up_apps[namespace_id][group_name] = {}
+        end
+
         for _, host in ipairs(data.hosts) do
-            local nodes = up_apps[service_info.service_name]
+            local nodes = up_apps[namespace_id]
+            [group_name][service_info.service_name]
```

Comment:
It will be better to merge it into one line for readability.

Reply:
done